Classes/Bookmark/BookmarkStore.swift

```diff
@@ -29,6 +33,7 @@ final class BookmarksStore {
 
     private var token: String
     private var _bookmarks: Set<BookmarkModel> = []
+    private var listeners: [BookmarksStoreListner] = []
```

Comment:
Retain cycle b/c listeners are stored strongly. You can view a couple ways to store many `weak` objects [here](https://github.com/rnystrom/GitHawk/blob/master/Local%20Pods/FlatCache/FlatCache/FlatCache.swift#L59) and [here (check out deleted code)](https://github.com/rnystrom/GitHawk/commit/5bae4ba89dc29daf1c10bd8b5e95006c5b04ab50#diff-d08adcb8b7b2ae5fc2e0f5abfaa92a42)